sql/src/main/java/io/crate/metadata/doc/DocTableInfo.java

```diff
@@ -97,6 +100,7 @@ public DocTableInfo(TableIdent ident,
                         boolean isAlias,
                         boolean hasAutoGeneratedPrimaryKey,
                         String[] concreteIndices,
```

Comment:
please double check the concreteIndices usages to make sure nothing has to be updated to use the openIndices instead